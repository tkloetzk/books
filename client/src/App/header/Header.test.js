import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';

describe('Header', () => {
  let props;
  let wrapper;

  beforeEach(() => {
    props = {
      children: 'Book Review Aggregator',
      scrollAction: 250,
    };
    wrapper = shallow(<Header {...props} />);
  });
  afterEach(() => {
    jest.clearAllMocks();
  });
  it('renders with no props', () => {
    wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
  it('renders with correct props', () => {
    wrapper = shallow(<Header {...props} />);
    expect(wrapper).toMatchSnapshot();
  });
});
