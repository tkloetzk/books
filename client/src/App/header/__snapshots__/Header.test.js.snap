// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders with correct props 1`] = `
<header
  className="header"
  style={
    Object {
      "height": undefined,
      "zIndex": 1,
    }
  }
>
  <div
    className="header-content display-4"
  >
    Book Review Aggregator
  </div>
</header>
`;

exports[`Header renders with no props 1`] = `
<header
  className="header"
  style={
    Object {
      "height": undefined,
      "zIndex": 1,
    }
  }
>
  <div
    className="header-content display-4"
  />
</header>
`;
